### PR TITLE
[skip ci] updated harbor topics with review comments

### PR DESCRIPTION
**Pull request by npakrasi**
-incorporated loc's comments for the harbor topics in app dev and install

### CODE REVIEW

**Comment by npakrasi** [comment]
LGTM

File: doc/user_doc/vic_app_dev/using_vic_with_harbor.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
 # Using vSphere Integrated Containers Engine with vSphere Integrated Containers Registry (Harbor)
 
-This example illustrates using a deployed Virtual Container Host (VCH) with Harbor as a private registry with the assumption that a VCH has been set up using either static IP or FQDN and standard docker has been updated with the certificate authority cert that can verify the deployed Harbor's server certificate. See [Deploying vSphere Integrated Container Engine with vSphere Integrated Containers Registry (Harbor)](deploy_vic_with_harbor.md) for more information on updating certificates.
+This example illustrates using a deployed Virtual Container Host (VCH) with Harbor as a private registry with the assumption that a VCH has been set up using either static IP or FQDN.  It also assumes there is access to standard Docker that has been updated with the CA certificate used by the Harbor instance.
```
Comment (sflxn) [inline doc/user_doc/vic_app_dev/using_vic_with_harbor.md]: You know what, I need to change this again.

```
that has been updated with the CA certificate used to sign the Harbor instance's server certificate and server private key.
```

**Comment by sflxn** [comment]
lgtm after you change that one sentence.

**Comment by npakrasi** [comment]
.

**Pull request merged**